### PR TITLE
New classifier

### PR DESCRIPTION
**Pull request by Katy-katy**
@lensonp  I also change the learning rate in this branch since now we do not have huge multiples class classifiers and tests should we OK. This new learning rate reduces randomness in the coef of the models (each time when we train the model with the same data and parameters we get about the same  errors and coefficients)

### CODE REVIEW

**Comment by lensonp** [comment]
@Katy-katy - I am now rebasing this branch on dev, and then I'll review it. If you work on this, fetch and reset first.

**Comment by lensonp** [comment]
@Katy-katy 

> I also change the learning rate in this branch...

We want to use learning_rate='constant', right?

**Comment by Katy-katy** [comment]
Ok - let put "constant" - it definitely works. later I will try "adaptive" - it also seams to work and with "adaptive" the training take less time.

**Comment by lensonp** [comment]
It looks like 'constant' takes so long that the CI test stalls...

**Comment by Katy-katy** [comment]
yea... let's use "adaptive"

**Comment by lensonp** [comment]
OK- I used the default, and it seems to work. I'll do this review tomorrow morning.

**Comment by Katy-katy** [comment]
@lensonp reviewing your pull request I realized that we do not save hyper parameters. I am going to fix it now in this branch

**Comment by lensonp** [comment]
@Katy-katy - Nice catch. Let me know when it's done, and I'll review it.

**Comment by Katy-katy** [comment]
@it is ready now

**Comment by Katy-katy** [comment]
@lensonp We still have the problem with the learning rate. I trained the models two times and got: diffuse model F1 0.71 and 0.61; disordered model F1 0.68 and 0.66; n_diffuse 0.73 and 0.69. So, tomorrow morning or about lunch time I will create a pull request with the learning rate that gives us a stable result

**Comment by lensonp** [comment]
@Katy-katy - Ok, sounds good. At least those F1 scores are fairly similar to one another.

The problem may not be the learning rate, though-- the group assignment method uses KMeans to rebalance groups, so the results are somewhat stochastic. I've investigated this, and the groups can actually be __very__ different from one KMeans to the next. One solution is to change KMeans(n_init=10) to KMeans(n_init=100) or higher, but obviously, this slows down the training pretty badly.

**Comment by Katy-katy** [comment]
@lensonp I got the stable result without changing KMeans(n_init=10). I train the models 4 times. I got: diffuse [0.69, 0.69, 0.69, 0.69], disordered  [0.69, 0.69, 0.69, 0.69], n_diffuse [0.67, 0.67, 0.66, 0.67]. On my machine, the training takes about 25 minutes.

**Comment by Katy-katy** [comment]
@lensonp  all tests passed in 367 sec in my machine
